javalib/src/main/scala/java/lang/Thread.scala

```diff
@@ -47,4 +47,23 @@ object Thread {
     currentThread.interruptedState = false
     ret
   }
+
+  def sleep(millis: scala.Long, nanos: scala.Int): Unit = {
+    import scala.scalanative.native._
+
+    def checkErrno() =
+      if (errno.errno == errno.EINTR)
+        throw new InterruptedException("Sleep was interrupted")
+
+    if (millis < 0) throw new IllegalArgumentException("millis must be >= 0")
+    else if (nanos < 0 || nanos > 999999)
+      throw new IllegalArgumentException("nanos value out of range")
+
+    val secs  = millis / 1000
+    val usecs = (millis % 1000) * 1000 + nanos / 1000
+    if (secs > 0 && unistd.sleep(secs.toUInt) != 0) checkErrno()
+    if (usecs > 0 && unistd.usleep(usecs) != 0) checkErrno()
```

Comment:
Hmm, unsigned int is not normally capable of holding an integer up ta 100 000, but max 65 535. See UINT_MAX: http://tigcc.ticalc.org/doc/limits.html#UINT_MAX Because of that I set it to Long.